pkg/generated/openapi/BUILD

```diff
@@ -74,6 +74,8 @@ openapi_library(
         "k8s.io/metrics/pkg/apis/external_metrics/v1beta1",
         "k8s.io/metrics/pkg/apis/metrics/v1alpha1",
         "k8s.io/metrics/pkg/apis/metrics/v1beta1",
+        "k8s.io/sample-apiserver/pkg/apis/wardle/v1alpha1",
+        "k8s.io/sample-apiserver/pkg/apis/wardle/v1beta1",
```

Comment:
these should not be here.

Reply:
@sttts I think that #63960 only applies to directories that start with underscore like [ _examples](https://github.com/kubernetes/kubernetes/tree/f8de7cea406a8d01799c4b4d40b892f3b38fa534/staging/src/k8s.io/code-generator/_examples) . I'll add the sample-apiserver to the skipped paths as well.

Reply:
👍🏼

Reply:
`.kazelcfg.json` with the content bellow seems to fix the issue. Is there any better way to do it?

```json
{
	"GoPrefix": "k8s.io/kubernetes",
	"SkippedPaths": [
		"^_.*",
		"/_",
		"^third_party/etcd.*",
		"k8s.io/sample-apiserver/.*"
	],
	"AddSourcesRules": true,
	"K8sOpenAPIGen": true
}
```